config/filter.d/ignorecommands/apache-fakegooglebot

```diff
@@ -26,7 +26,8 @@ def is_googlebot(ip):
     from fail2ban.server.filter import DNSUtils
 
     host = DNSUtils.ipToName(ip)
-    sys.exit(0 if (host and re.match('crawl-.*\.googlebot\.com', host)) else 1)
```

Comment:
Please don't execute dnsToIp, if already the host does not match googlebot (99.99% of all cases).
I mean rewrite it without sugar:
```python
if host and re.match('crawl-.*\.googlebot\.com', host):
  sys.exit(0)
host_ips = DNSUtils.dnsToIp(host)
sys.exit(0 if ip in host_ips else 1)
```